buildSrc/src/main/kotlin/com/frybits/harmony/gradle/FrybitsApplicationPlugin.kt

```diff
@@ -33,10 +33,10 @@ import org.jetbrains.kotlin.gradle.plugin.KotlinAndroidPluginWrapper
 
 class FrybitsApplicationPlugin: Plugin<Project> {
 
-    override fun apply(target: Project) {
-        target.applyAppPlugins()
+    override fun apply(target: Project) = target.run {
```

Comment:
Question - This shouldn't be any functional change, correct?

Reply:
No, this a minor change for the build logic and is not part of Harmony.